src/App.js

```diff
@@ -103,7 +103,7 @@ import AnimalsRandomDog from "./api/animals/AnimalsRandomDog";
 import AnimalsShibeOnline from "./api/animals/AnimalsShibeOnline";
 import GeoCodingGetTheData from "./api/geocoding/GeoCodingGetTheData";
 import GeoCodingCountry from "./api/geocoding/GeoCodingCountry";
-import Catfacts from "./api/animals/AnimalCatFacts";
+import AnimalCatFacts from "./api/animals/AnimalCatFacts";

```

Comment:
Animals

Reply:
Sorry my Bad I have made the necessary changes